torchvision/models/optical_flow/_raft/utils.py

```diff
@@ -0,0 +1,42 @@
+import torch
+import torch.nn.functional as F
+
+
+def grid_sample(img, absolute_grid, *args, **kwargs):
+    """Same as torch's grid_sample, with absolute pixel coordinates instead of normalized coordinates."""
```

Comment:
Food for thought: would it be a significant limitation if we were to return the flows in normalized coordinates, and perform all computations in normalized coordinates?

Or is it standard that flow images are in absolute coordinates?

Some of the benefits of keeping it in relative coordinates is that you don't need to multiply by the scaling factor when upsampling an image.